cmd/crates/soroban-spec-typescript/src/lib.rs

```diff
@@ -266,7 +255,7 @@ pub fn entry_to_method_type(entry: &Entry) -> String {
         Entry::TupleStruct { doc, name, fields } => {
             let docs = doc_to_ts_doc(doc, None, 0);
             let fields = fields.iter().map(type_to_ts).join(",  ");
-            format!("{docs}export type {name} = readonly [{fields}];")
+            format!("{docs}export type {name} = readonly [{fields}];\n")
```

Comment:
This fixes an inconsequential formatting error in the generated TS, since I noticed it while I was here and it was easy to fix